client/my-sites/invites/invite-accept/index.jsx

```diff
@@ -82,8 +82,14 @@ export default React.createClass( {
 		}
 		debug( 'Rendering invite' );
 		return user.get()
-			? <LoggedIn { ...this.state.invite } redirectTo={ this.getRedirectTo() } />
-			: <LoggedOut { ...this.state.invite } redirectTo={ this.getRedirectTo() } />;
+			? <LoggedIn
+				{ ...this.state.invite }
+				redirectTo={ this.getRedirectTo() } />
+			: <LoggedOut
+				{ ...this.state.invite }
+				redirectTo={ this.getRedirectTo() }
+				subscriptionActivationKey={ this.props.activation_key }
```

Comment:
I meant here, but now that you mention, can't we get rid of at least one of the 3 keys?